src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -221,6 +227,25 @@ UserImportResult importUsers(UserImportRequest request) throws FirebaseAuthExcep
     return new UserImportResult(request.getUsersCount(), response);
   }
 
+  // TODO(micahstairs): Write unit tests for this method.
+  ListTenantsResponse listTenants(int maxResults, String pageToken) throws FirebaseAuthException {
+    ImmutableMap.Builder<String, Object> builder = ImmutableMap.<String, Object>builder()
+        .put("maxResults", maxResults);
+    if (pageToken != null) {
+      checkArgument(!pageToken.equals(
+          ListTenantsPage.END_OF_LIST), "invalid end of list page token");
+      builder.put("nextPageToken", pageToken);
```

Comment:
I think the query params are called `pageSize` and `pageToken`. Verify and correct if necessary.

Reply:
You are correct.